runtime/runtime/src/congestion_control.rs

```diff
@@ -461,9 +527,28 @@ impl DelayedReceiptQueueWrapper {
     }
 }
 
-pub(crate) fn receipt_size(receipt: &Receipt) -> Result<usize, IntegerOverflowError> {
-    // `borsh::object_length` may only fail when the total size overflows u32
-    borsh::object_length(&receipt).map_err(|_| IntegerOverflowError)
+/// Get the receipt size from the receipt that was retrieved from the state.
+/// If it is a [Receipt], the size will be computed.
+/// If it s the [StateStoredReceipt], the size will be read from the metadata.
+pub(crate) fn receipt_congestion_size(
```

Comment:
nit: `receipt_size ` was fine IMO, and I'll need it for bandwidth limits as well, so I'd prefer the name to not have `congestion` in it.

Reply:
That's true, eh maybe I can make a separate function for my purposes, up to you.